fontir/src/orchestration.rs

```diff
@@ -29,15 +29,15 @@ bitflags! {
         // If set, a composite that references another composite will replace that composite with the
         // glyph(s) it references until only simple (contour) glyphs are referenced
         const FLATTEN_COMPONENTS = 0b00001000;
+        const ALLOW_COMPONENT_TRANSFORM = 0b00010000;
         // If set a files reporting on timing will be emitted to disk
-        const EMIT_TIMING = 0b00010000;
+        const EMIT_TIMING = 0b00100000;
```

Comment:
maybe we should make Flags a u64, looks like we may run out of empty slots soon

Reply:
Agreed but as best I can tell we can defer that until we actually hit it because nothing outside fontc relies on these flags.